linkerd/stack/src/box_response.rs

```diff
@@ -0,0 +1,43 @@
+use std::{
+    future::Future,
+    pin::Pin,
+    task::{Context, Poll},
+};
+
+/// A middleware that boxes *just* the response future from an inner service,
+/// without erasing the service's type (and its trait impls, such as `Clone`).
+///
+/// This is primarily useful when a service's `Future` type is not `Unpin` and
+/// must be boxed.
+#[derive(Clone, Debug)]
+pub struct BoxResponse<T>(T);
```

Comment:
this was renamed to `BoxFuture` before merging #826, so I think this file is now a duplicate and is unused...can we just remove this?